edge-util/src/Microsoft.Azure.Devices.Edge.Util/uds/HttpRequestResponseSerializer.cs

```diff
@@ -99,17 +100,28 @@ async Task SetHeadersAndContent(HttpResponseMessage httpResponse, HttpBufferedSt
                 bool headerAdded = httpResponse.Headers.TryAddWithoutValidation(headerName, headerValue);
                 if (!headerAdded)
                 {
-                    if (string.Equals(headerName, ContentLengthHeaderName, StringComparison.InvariantCultureIgnoreCase))
-                    {
-                        if (!long.TryParse(headerValue, out long contentLength))
-                        {
-                            throw new HttpRequestException($"Header value is invalid for {headerName}.");
-                        }
+                    contentHeaders.Add(headerName, headerValue);
+                }
+            }
+
+            bool isChunked = httpResponse.Headers.TransferEncodingChunked.HasValue
```

Comment:
Does hyper set the chunked encoding header?

Reply:
Yes, it does.